src/util/MongoBinaryDownload.js

```diff
@@ -91,13 +91,22 @@ export default class MongoBinaryDownload {
   }
 
   async checkMd5(mongoDBArchiveMd5: string, mongoDBArchive: string) {
```

Comment:
I know it's not you that created this function but I don't understand why this function is declared async since there are no asynchronous operation in it.

Reply:
@AJRdev me too. Sure, I'll change it

Reply:
@dimitriylol Ok cool :) I think it will impact all your tests that expect a Promise from this function.

Reply:
Async `checkMd5` is made for compatibility with other download methods. So better to keep it as is.

Better to rewrite implementation in async way. But let it keep for another PR.

Reply:
@nodkz if so, I need to remove my last commit